### PR TITLE
Updating README and CHANGELOG for public preview of Cosmos Encryption

### PR DESCRIPTION
**Pull request by simplynaveen20**
Updating README and CHANGELOG for public preview of Cosmos Encryption

Also upgraded to 0.2.1 version of cryptography

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
-
+## 1.0.0-beta.5 (2021-05-20)
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]: Version headers should start with 3 hashes - to make it H2 title for Microsoft docs. 
```suggestion
### 1.0.0-beta.5 (2021-05-20)
```
Please update all of these versions to H2 title by adding 1 more # to them.

File: sdk/cosmos/azure-cosmos-encryption/README.md [hunk 2]
```diff
@@ -9,26 +9,180 @@ The encryption plugin is under development and not ready to be consumed yet.
...
+
+Also add a log4j config.
+
+```properties
+# this is a sample log4j configuration
+
+# Set root logger level to INFO and its only appender to A1.
+log4j.rootLogger=INFO, A1
+
+log4j.category.com.azure.cosmos=INFO
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-encryption/README.md:171]: need to add logging for encryption as well here, if we have any logging in encryption project. 
```suggestion
log4j.category.com.azure.cosmos.encryption=INFO
```

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: sdk/cosmos/azure-cosmos-encryption/README.md (hunk 2 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/README.md:174]: path log4j.category.com.azure.cosmos will cover all the classes under it including cosmos sdk and cosmos encryption sdk

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md (hunk 1 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md]: done

File: sdk/cosmos/azure-cosmos-encryption/README.md [hunk 3]
```diff
@@ -44,13 +201,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
-[source_code]: src
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cosmos/azure-cosmos/src
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-encryption/README.md:204]: nit - we don't need this link.

**Review by kushagraThapar — APPROVED** [review]
LGTM except one small minor change

File: sdk/cosmos/azure-cosmos-encryption/README.md (hunk 3 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/README.md]: done , removed other non used url as well

**Pull request merged**